packages/web/src/components/cell/createCell.tsx

```diff
@@ -0,0 +1,179 @@
+import { getOperationName } from '../../graphql'
+/**
+ * This is part of how we let users swap out their GraphQL client while staying compatible with Cells.
+ */
+import { useQuery } from '../GraphQLHooksProvider'
+
+import { useCellCacheContext } from './CellCacheContext'
+import { CreateCellProps } from './cellTypes'
+import { createSuspendingCell } from './createSuspendingCell'
+import { isDataEmpty } from './isCellEmpty'
+
+// 👇 Note how we switch which cell factory to use!
+export const createCell = RWJS_ENV.RWJS_EXP_STREAMING_SSR
+  ? createSuspendingCell
+  : createNonSuspendingCell
+
+/**
+ * Creates a Cell out of a GraphQL query and components that track to its lifecycle.
+ */
+function createNonSuspendingCell<
```

Comment:
This function is actually unchanged. Just renames/moved stuff around. Common stuff moved to `cellTypes` and `isCellEmpty`